models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/OllamaChatModel.java

```diff
@@ -201,53 +202,43 @@ public Flux<ChatResponse> stream(Prompt prompt) {
 	 */
 	OllamaApi.ChatRequest ollamaChatRequest(Prompt prompt, boolean stream) {
 
-		List<OllamaApi.Message> ollamaMessages = prompt.getInstructions()
-			.stream()
-			.filter(message -> message.getMessageType() == MessageType.USER
```

Comment:
This diff looks really weird, but the only change I made was removing the "filter()" expression since we now accept all message types, so there's no filter needed.

Reply:
Looks like the Spring Formatter plugin made formatting changes after removing the filter() part.